SSE is enabled by defalut on all new storage accounts

JackStromberg opened the pull request:
Please see the following announcement: https://azure.microsoft.com/en-us/blog/announcing-default-encryption-for-azure-blobs-files-table-and-queue-storage/

azurecla commented:
Hi __@jaymathe__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (jaymathe). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

JackStromberg committed:
SSE is enabled by defalut on all new storage accounts

Please see the following announcement: https://azure.microsoft.com/en-us/blog/announcing-default-encryption-for-azure-blobs-files-table-and-queue-storage/

tamram commented on articles/storage/common/storage-service-encryption.md:
```diff
@@ -156,7 +156,7 @@ A: Not at this time; the keys are fully managed by Microsoft.
 **Q: Is SSE enabled by default when I create a new storage account?**
 
-A: SSE is not enabled by default; you can use the Azure portal to enable it. You can also programmatically enable this feature using the Storage Resource Provider REST API.
+A: As of August 31, 2017 encryption will be enabled default using Microsoft Managed Keys for all data written to Azure services (Blob, File, Table and Queue storage), for all storage accounts (Azure Resource Manager and Classic storage accounts), both new and existing.
```
@JackStromberg - Can you make a couple of grammatical fixes?

As of August 31, 2017, encryption is enabled by default using Microsoft Managed Keys for all data written to Azure Storage (Blob, File, Table and Queue storage), and for all storage accounts (Azure Resource Manager and Classic storage accounts), both new and existing.

JackStromberg committed:
Change to grammar

JackStromberg committed:
Updated SSE default verbiage

GitHubber17 committed:
Update storage-service-encryption.md
